ports/servo/glutin_app/window.rs

```diff
@@ -1122,12 +1042,11 @@ impl WindowMethods for Window {
     fn set_favicon(&self, _: BrowserId, _: ServoUrl) {
     }
 
-    fn prepare_for_composite(&self, _width: Length<u32, DevicePixel>, _height: Length<u32, DevicePixel>) -> bool {
-        true
-    }
-
     /// Helper function to handle keyboard events.
-    fn handle_key(&self, _: Option<BrowserId>, ch: Option<char>, key: Key, mods: constellation_msg::KeyModifiers) {
+    fn handle_key(&self, _: Option<BrowserId>, ch: Option<char>, key: Key, state: KeyState, mods: constellation_msg::KeyModifiers) {
+        if state == KeyState::Pressed {
```

Comment:
Why are we ignoring pressed keys here?

Reply:
Only pressed keys are sent back to the embedder. It's the original behavior: https://github.com/servo/servo/blob/bedeec0cf54011e3d15bd50ba32b6b422958b483/components/servo/lib.rs#L428 introduced here: https://github.com/servo/servo/pull/4822/commits/37cb876f4ea6c39390c359be068f352e2d47d8a0